cmd/utils/flags.go

```diff
@@ -254,6 +254,10 @@ var (
 		Name:  "ulc.onlyannounce",
 		Usage: "Ultra light server sends announcements only",
 	}
+	LightPruningFlag = cli.BoolFlag{
+		Name:  "light.pruning",
```

Comment:
IMHO this should be `--light.prune`. That seems to indicate a request to prune light client data. The `--light.pruning` seems to need a second parameter.